CHANGELOG.md

```diff
@@ -42,7 +73,7 @@ All notable changes to this project will be documented in this file. The format
 
 ### Fixed
 
-- \(FM-8662\) Correct  in manifests/mod/ssl.pp for SLES 11 [\#1963](https://github.com/puppetlabs/puppetlabs-apache/pull/1963) ([cmccrisken-puppet](https://github.com/cmccrisken-puppet))
+- \(FM-8662\) Correct  in manifests/mod/ssl.pp for SLES 11 [\#1963](https://github.com/puppetlabs/puppetlabs-apache/pull/1963) ([sanfrancrisko](https://github.com/sanfrancrisko))
```

Comment:
Noticed this typo and fixed it. No biggie, but if we wanted to, we could re-run the GHCG and get that fix